hack/lib/servicemesh.bash

```diff
@@ -1,24 +1,6 @@
 #!/usr/bin/env bash
 
-function ensure_service_mesh_installed {
-  local istio_hostname istio_ip
-  logger.info 'Checking if Service Mesh is installed...'
-
-  if oc get namespaces istio-system > /dev/null 2>&1; then
-    istio_hostname=$(kubectl get svc -n istio-system istio-ingressgateway -o jsonpath="{.status.loadBalancer.ingress[0].hostname}")
-    istio_ip=$(resolve_hostname "${istio_hostname}")
-    if [[ "${istio_ip}" != "" ]]; then
-      logger.success 'Service Mesh seems operational. Skipping installation.'
-      setup_service_mesh_member_roll
-      return 0
-    fi
-  fi
-  install_service_mesh
-}
-
-function install_service_mesh {
-  logger.info 'Installing ServiceMesh'
-
+function install_service_mesh_operator {
```

Comment:
I would expect only the `CatalogSourceConfig` is needed in this function. All of the Subscriptions can be deleted if the CRD dependencies in the CSV's are correct, I think.

Reply:
Openshift 4.1 needs the manual subscription stuff and this repo is currently only setup against 4.1 CI.

Reply:
On 4.2 this entire function is no longer needed. See https://github.com/openshift/knative-serving/pull/298 for how that looks like.